### PR TITLE
feat: subscribe btn

### PR DESCRIPTION
**Pull request by YoshkiMatryoshki**
call subscribe\unsubscribe from ui. button visible only if user is logged in.

there is no popovers to notify user for this button at the moment

### CODE REVIEW

File: src/components/svg_image.rs [hunk 1]
```diff
@@ -97,3 +97,21 @@ pub fn counterclockwise_img() -> Html {
         </svg>
     }
 }
+
+#[function_component(UnsubscribedImg)]
+pub fn search_img() -> Html {
```
Comment (tikitko) [inline src/components/svg_image.rs:102]:
```suggestion
pub fn unsubscribed_img() -> Html {
```

File: src/components/svg_image.rs [hunk 2]
```diff
@@ -97,3 +97,21 @@ pub fn counterclockwise_img() -> Html {
...
+    html! {
+        <svg aria-label="Отписка" xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-bell-slash-fill" viewBox="0 0 16 16">
+            <path d="M5.164 14H15c-1.5-1-2-5.902-2-7 0-.264-.02-.523-.06-.776zm6.288-10.617A4.988 4.988 0 0 0 8.995 2.1a1 1 0 1 0-1.99 0A5.002 5.002 0 0 0 3 7c0 .898-.335 4.342-1.278 6.113l9.73-9.73M10 15a2 2 0 1 1-4 0zm-9.375.625a.53.53 0 0 0 .75.75l14.75-14.75a.53.53 0 0 0-.75-.75z"/>
+        </svg>
+    }
+}
+
+#[function_component(SubscribedImg)]
+pub fn search_img() -> Html {
```
Comment (tikitko) [inline src/components/svg_image.rs:111]:
```suggestion
pub fn subscribed_img() -> Html {
```

File: src/components/subscribe_button.rs [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        html! {}
+    } else {
+        html! {
+            <button
+            type="button"
+            class={ classes!("btn","btn-light", if author.is_none() { "invisible"} else {"visible"}) }
+            disabled={ *in_progress }
+            { onclick }
+            >
+            if *is_subscribed {
```
Comment (tikitko) [inline src/components/subscribe_button.rs:96]: inner content padding

**Review by tikitko — CHANGES_REQUESTED** [review]

**Comment by tikitko** [comment]
plz check jobs.
Plz fit SSR requirements....

File: src/components/svg_image.rs [hunk 4]
```diff
@@ -97,3 +97,21 @@ pub fn counterclockwise_img() -> Html {
         </svg>
     }
 }
+
+#[function_component(UnsubscribedImg)]
+pub fn search_img() -> Html {
+    html! {
+        <svg aria-label="Отписка" xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-bell-slash-fill" viewBox="0 0 16 16">
```
Comment (tikitko) [inline src/components/svg_image.rs:104]:
```suggestion
        <svg aria-label="DENIS KONCH" xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-bell-slash-fill" viewBox="0 0 16 16">
```

File: src/components/svg_image.rs [hunk 5]
```diff
@@ -97,3 +97,21 @@ pub fn counterclockwise_img() -> Html {
...
+            <path d="M5.164 14H15c-1.5-1-2-5.902-2-7 0-.264-.02-.523-.06-.776zm6.288-10.617A4.988 4.988 0 0 0 8.995 2.1a1 1 0 1 0-1.99 0A5.002 5.002 0 0 0 3 7c0 .898-.335 4.342-1.278 6.113l9.73-9.73M10 15a2 2 0 1 1-4 0zm-9.375.625a.53.53 0 0 0 .75.75l14.75-14.75a.53.53 0 0 0-.75-.75z"/>
+        </svg>
+    }
+}
+
+#[function_component(SubscribedImg)]
+pub fn search_img() -> Html {
+    html! {
+        <svg aria-label="Подписька" xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-bell-fill" viewBox="0 0 16 16">
```
Comment (tikitko) [inline src/components/svg_image.rs:113]:
```suggestion
        <svg aria-label="DENIS KONCH" xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-bell-fill" viewBox="0 0 16 16">
```

File: src/components/svg_image.rs (hunk 2 above)
Comment (tikitko) [reply src/components/svg_image.rs:111]: + we using generic names for icons, for example `bell`

File: src/components/svg_image.rs (hunk 5 above)
Comment (tikitko) [inline src/components/svg_image.rs:113]: without aria-label

Comment (YoshkiMatryoshki) [reply src/components/svg_image.rs]: done

Comment (YoshkiMatryoshki) [reply src/components/svg_image.rs]: done

File: src/components/subscribe_button.rs (hunk 3 above)
Comment (YoshkiMatryoshki) [reply src/components/subscribe_button.rs]: done

File: src/components/svg_image.rs (hunk 2 above)
Comment (YoshkiMatryoshki) [reply src/components/svg_image.rs]: done

File: src/components/svg_image.rs (hunk 1 above)
Comment (YoshkiMatryoshki) [reply src/components/svg_image.rs]: done

**Comment by YoshkiMatryoshki** [comment]
> plz check jobs. Plz fit SSR requirements....

WUT?

**Review by tikitko — APPROVED** [review]

**Pull request merged**